packages/sw-build-webpack-plugin/index.js

```diff
@@ -0,0 +1,70 @@
+const swBuild = require('../sw-build/src/');
+const path = require('path');
+const SERVICE_WORKER_NAME = 'sw.js';
+
+/**
+ * Use the instance of this in the plugins array of the webpack config.
+ * @example
```

Comment:
Is it worth including an example and explanation of how to use the inject file or should we leave that up to some other docs?

cc @jpmedley